README.md

```diff
@@ -338,6 +339,43 @@ List of whitepapers to the described projects.
 * [Hyper Oracle](https://mirror.xyz/hyperoracleblog.eth/qbefsToFgFxBZBocwlkX-HXbpeUzZiv2UB5CmxcaFTM)
 * [QED](https://www.qedprotocol.com/)
 
+### Benchmarks
+
+[🏋️‍♀️ ZK Bench](https://zkbench.dev) - open source, continuous benchmarks for popular zk implementations
+
```

Comment:
Thank you for your PR :) `zkbench.dev` looks great, keep building and testing!
Link to your website is definitely in the right place, but please remove the table, as you maintain it on the website.

Reply:
Makes sense, done!